contributors/guide/release-notes.md

```diff
@@ -10,7 +10,7 @@ output format changes. Release notes are one of the most important reference
 points for users about to install or upgrade to a particular release of
 Kubernetes.
 
-## Does my pull request need a release note?
+## Does my pull request needs a release note?
```

Comment:
The initial text is correct, there's no need for an `s` here.

Reply:
Yep, reverted the change :+1: